lib/administrate/custom_dashboard.rb

```diff
@@ -0,0 +1,15 @@
+module Administrate
+  class CustomDashboard
+    include Administrate
+
+    class << self
+      def resource_name(_opts)
+        named_resource.pluralize.titleize
+      end
+
+      def resource(resource_name)
+        define_singleton_method(:named_resource) { resource_name }
```

Comment:
What do you think about inferring the resource_name from its class? 

```ruby
resource_name || self.name.remove('Dashboard')
```

Reply:
I don't have a strong opinion on this; can you see any downsides of taking this approach compared to what we're doing?

Reply:
Just in the sake of time - since we want to merge it in tomorrow - let's skip this change